app/controllers/internal/configs_controller.rb

```diff
@@ -135,6 +135,8 @@ def image_params
 
   def onboarding_params
```

Comment:
We should also add a test to the spec⁩/⁨requests⁩/internal⁩/configs_spec to test the updating of the       onboarding_logo_image and onboarding_background_image.

Reply:
✅